CONTRIBUTING.md

```diff
@@ -322,6 +324,38 @@ doesn't need to wait. A Pull Request may well take longer to be
 merged in. All these precautions are important because Node.js is
 widely used, so don't be discouraged!
 
+### Backporting to Current and LTS Branches
+
+Before backporting or contributing to the LTS releases make sure you are
+familiar with the LTS information and schedules at
+https://github.com/nodejs/LTS#lts-schedule
+
+Items which have already made it from master will be candidates for the
+"current" release branch(es) and can then also become candidates for
+inclusion into the "Active LTS" branch(es), especially if they are bug
+fixes.  Any change can be considered for LTS by applying the appropriate
```

Comment:
If we remove that we lose the information about the sequence required for things to get into lts - since in general we don't want to give people the impression they could get straight into LTS releases without going through master  and current, although I admit I'm not too happy about the flow of that sentence.